src/main/java/com/kodilla/ecommercee/domain/dto/OrderItemDto.java

```diff
@@ -15,4 +15,5 @@ public class OrderItemDto {
     private BigDecimal price;
     private Long productsId;
     private Long orderId;
+    private Long quantity;
```

Comment:
Due to ERD Diagram the name of new field is "productQuantity" not `quantity`. 
And it should be `int`, not `Long` - you used even `int` above in `ProductsInCart`, and I used that in base implementation of entity OrderItem. Maybe you want to change too much in one commit? :)

Reply:
Resolved